src/services/api.ts

```diff
@@ -27,10 +27,11 @@ export async function fetchComicContent(
 
 // Call the API to get a specified comic's chapters based on the comic's hid.
 export async function fetchComicChapters(
-  comicHid: string
+  comicHid: string,
+  chapterCount: number
 ): Promise<ComicChapters> {
   const response = await axios.get(
```

Comment:
Also your response here is not typed

Reply:
I will chekc everything in my improvements and fixes branch.